gear-programs/vft-treasury/src/services/error.rs

```diff
@@ -0,0 +1,32 @@
+use sails_rs::prelude::*;
+
+#[derive(Debug, Encode, Decode, TypeInfo, Clone, PartialEq, Eq)]
+pub enum Error {
+    SendFailure,
+    ReplyFailure,
+    RequestToBuiltinSend,
+    RequestToBuiltinReply,
+    BuiltinDecode,
+    PayloadSize,
+    ReplyTimeout,
+    DuplicateAddressMapping,
+    NoCorrespondingEthAddress,
+    ReplyHook,
+    MessageNotFound,
+    InvalidMessageStatus,
+    MessageFailed,
+    BridgeBuiltinMessageFailed,
+    TokensRefunded,
+    NotEthClient,
+    NotAdmin,
+    NotBridgingClient,
+    NotEnoughGas,
+    TransferFailed,
+    TransferTokensDecode,
+    TokensTransferFailure,
+    RequestToGateWayDecode,
+    TransactionFailure,
+    FailureInVftGateway,
+    GatewayMessageProcessingFailed,
+    TransferTokensFailed,
+}
```

Comment:
Some errors are not used in the contract. Please either remove or use the unused ones.